compiler/aarch64/codegen/ARM64Ops.ops

```diff
@@ -0,0 +1,1213 @@
+!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+!! Copyright (c) 2018, 2018 IBM Corp. and others
+!!
+!! This program and the accompanying materials are made available under
+!! the terms of the Eclipse Public License 2.0 which accompanies this
+!! distribution and is available at http://eclipse.org/legal/epl-2.0
+!! or the Apache License, Version 2.0 which accompanies this distribution
+!! and is available at https://www.apache.org/licenses/LICENSE-2.0.
+!!
+!! This Source Code may also be made available under the following Secondary
+!! Licenses when the conditions for such availability set forth in the
+!! Eclipse Public License, v. 2.0 are satisfied: GNU General Public License,
+!! version 2 with the GNU Classpath Exception [1] and GNU General Public
+!! License, version 2 with the OpenJDK Assembly Exception [2].
+!!
+!! [1] https://www.gnu.org/software/classpath/license.html
+!! [2] http://openjdk.java.net/legal/assembly-exception.html
+!!
+!! SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+!		Opcode      	BINARY  	Properties               	// {Specific} <variant> [comments]
+! UNALLOCATED
+		bad         	00000000	-                        	// [ invalid operation ]
```

Comment:
Devin was talking about the use of 'bad' today for the 'break on entry' capability. If there is an instruction that works cleanly with gdb as a break-point, that gdb knows it can step past with 'continue' then perhaps we could use that here. I wonder if 'brk' does that?

Reply:
Good question I'm not sure. We'll have to look into this.